gen/nvml/init.go

```diff
@@ -88,6 +88,9 @@ var nvmlDeviceGetNvLinkRemotePciInfo = nvmlDeviceGetNvLinkRemotePciInfo_v1
 var nvmlDeviceRemoveGpu = nvmlDeviceRemoveGpu_v1
 var nvmlDeviceGetGridLicensableFeatures = nvmlDeviceGetGridLicensableFeatures_v1
 var nvmlEventSetWait = nvmlEventSetWait_v1
+var nvmlDeviceGetAttributes = nvmlDeviceGetAttributes_v1
```

Comment:
I think we could generate this and the code for `updateVersionedSymbols()` from the header file directly. The pattern is quite straight-forward.

Reply:
Not critical for this review though.